logs_api/lib/opentelemetry/logs/severity_number.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module Logs
```

Comment:
@khushijain21 - Could you nest the `Logs` module under the `OpenTelemetry` module?

```ruby
module OpenTelemetry
  module Logs
    class SeverityNumber
      # ...
    end
  end
end
```

I think I did a poor job of explaining the nesting earlier today. I'll try to find a blog post that can describe things better.